packages/transport/src/modules/sync/externalEventProcessors/RelationshipCreatedOrChangedExternalEventProcessor.ts

```diff
@@ -3,10 +3,10 @@ import { Relationship } from "../../relationships/local/Relationship";
 import { BackboneExternalEvent } from "../backbone/BackboneExternalEvent";
 import { ExternalEventProcessor } from "./ExternalEventProcessor";
 
-export class RelationshipChangeCompletedExternalEventProcessor extends ExternalEventProcessor {
+export class RelationshipCreatedOrChangedExternalEventProcessor extends ExternalEventProcessor {
     public override async execute(externalEvent: BackboneExternalEvent): Promise<Relationship | undefined> {
-        const payload = externalEvent.payload as { changeId: string };
-        const relationship = await this.accountController.relationships.applyChangeById(payload.changeId);
+        const payload = externalEvent.payload as { relationshipId: string };
+        const relationship = await this.accountController.relationships.applyIncomingEvent(payload.relationshipId);
```

Comment:
I think we should rename `applyIncomingEvent` to `applyRelationshipStatusChangedEvent`